runtime/pom.xml

```diff
@@ -115,6 +123,18 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
These two test dependencies can be removed it seems.

Reply:
The dependencies are still present. Please remove it after removing RxJava imports from `MutinyReactiveResultSetTest`.